src/main.cpp

```diff
@@ -1,8 +1,7 @@
 #include "aldente.h"
 
 int main()
-{
-    Aldente grandma;
-    grandma.go();
+{    
+	Aldente::aldente->go();
```

Comment:
What's the motivation of ditching the local objects for these static pointers?  It makes it much harder to manage our memory...